### PR TITLE
PASMA-52: Front End - Landing Page

### PR DESCRIPTION
**Pull request by PistolRcks**
Makes a nice landing page containing the "Create Account" and "Login" buttons.

### CODE REVIEW

**Review by doranj5121 — APPROVED** [review]
Good clean up

File: client/dataHelper.js [hunk 1]
```diff
@@ -69,70 +69,79 @@ export async function sendUpdatedPassword (token, oldPass, newPass) {
...
-    
-    if(response.status === 401 || response.status === 500) {
-        window.alert((await response.text()).toString())
-        setIsFormDisabled(false)
-    }
-    else if(response.status === 200) {
-        window.alert("Post created successfully!\nPost id: " + (await response.text()))
+        body: JSON.stringify(newPost),
+    });
+
```
Comment (KarstenRoland) [inline client/dataHelper.js:134]: Change this to respond only to code 200 and show alert window for all other codes

File: client/dataHelper.js [hunk 2]
```diff
@@ -69,70 +69,79 @@ export async function sendUpdatedPassword (token, oldPass, newPass) {
...
+        body: JSON.stringify({ token: token }),
+    });
+    if (
+        getStockImages.status === 400 ||
+        getStockImages.status === 401 ||
+        getStockImages.status === 500
+    ) {
+        window.alert((await getStockImages.text()).toString());
+    } else if (getStockImages.status === 200) {
+        return JSON.parse(await getStockImages.text());
```
Comment (KarstenRoland) [inline client/dataHelper.js:116]: Change this to respond only to code 200 and show a window alert for all other response codes.

File: client/dataHelper.js [hunk 3]
```diff
@@ -69,70 +69,79 @@ export async function sendUpdatedPassword (token, oldPass, newPass) {
...
+        body: JSON.stringify({ token: token }),
+    });
+    if (
+        getPhrases.status === 400 ||
+        getPhrases.status === 401 ||
+        getPhrases.status === 500
+    ) {
+        window.alert((await getPhrases.text()).toString());
+    } else if (getPhrases.status === 200) {
+        return JSON.parse(await getPhrases.text());
```
Comment (KarstenRoland) [inline client/dataHelper.js:91]: Change this to respond only to code 200 and show a window alert for all other response codes.

**Pull request merged**